github/repos_projects.go

```diff
@@ -0,0 +1,461 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+)
+
+// Project represents a GitHub Project.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/projects/
+type Project struct {
+	ID        *int       `json:"id,omitempty"`
+	URL       *string    `json:"url,omitempty"`
+	OwnerURL  *string    `json:"owner_url,omitempty"`
+	Name      *string    `json:"name,omitempty"`
+	Body      *string    `json:"body,omitempty"`
+	Number    *int       `json:"number,omitempty"`
+	CreatedAt *Timestamp `json:"created_at,omitempty"`
+	UpdatedAt *Timestamp `json:"updated_at,omitempty"`
+
+	// The User object that generated the project.
+	Creator *User `json:"creator,omitempty"`
+}
+
+func (p Project) String() string {
+	return Stringify(p)
+}
+
+// ListProjects lists the projects for a repo.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/projects/#list-projects
+func (s *RepositoriesService) ListProjects(owner, repo string, opt *ListOptions) ([]*Project, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/projects", owner, repo)
+	u, err := addOptions(u, opt)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeProjectsPreview)
+
+	projects := []*Project{}
+	resp, err := s.client.Do(req, &projects)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return projects, resp, err
+}
+
+// GetProject gets a GitHub Project for a repo.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/projects/#list-a-project
+func (s *RepositoriesService) GetProject(owner, repo string, number int) (*Project, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/projects/%v", owner, repo, number)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeProjectsPreview)
+
+	project := &Project{}
+	resp, err := s.client.Do(req, project)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return project, resp, err
+}
+
+// ProjectInfo specifies the parameters to the
+// RepositoriesService.CreateProject and RepositoriesService.UpdateProject methods.
+type ProjectInfo struct {
```

Comment:
All other similar structs (that are used as last argument to the method in order to provide additional fields) are usually named with "Options" suffix in this library:

- https://godoc.org/github.com/google/go-github/github#ActivityListStarredOptions
- https://godoc.org/github.com/google/go-github/github#CommitsListOptions
- https://godoc.org/github.com/google/go-github/github#IssueListByRepoOptions
- https://godoc.org/github.com/google/go-github/github#RepositoryContentFileOptions
- ...

There aren't any with an Info suffix like in this PR.

Did you choose to use Info instead of Options on purpose? If so, what was the reason?